CIP - ?/README.md

```diff
@@ -0,0 +1,120 @@
+
+Raising a0 will make pledge benefit more noticeable, but this would mostly benefit the small handful of corporations that have pledged to saturation and would only barely effect the majority. It would not change the gap between the founders and everybody else. We need a change that will benefit the most people.
+
+The solutions in this CIP will solve all of the above mentioned problems. It will exponentially benefit more people/entities than in will detriment.
+
+
+## Solution:
+
+Change the minPoolCost from 340 ADA/epoch to 10 ADA/epoch and raise K from 500 to 2000.
+
+
+## Specification:
+
+"minPoolCost": 10000000
+"stakePoolTargetNum": 2000
+
+
+## Rationale:
+
+With minPoolCost set at 340 ADA/epoch the tipping point is 10M ADA. By lowering it to 10 ADA/epoch the tipping point drops to 1.75M {fig.4}. Now any pool with ≥ 1.75M pledge will no longer be competing with a sybille attacker for their first delegators. This tipping_point was chosen because it represents an upper-middle level of pledge within the real set of pools. An entrepreneur is far more likely to find 1.75M ADA investor over a 10M ADA investor. 
+
+As previously mentioned, there was no minPoolCost in {ref.2}. We have treated the mainnet like a testnet by deploying an untested, unsimulated, unproven method of sybille protection. 2 years is a generous test period. Time to evaluate the data and tweak these parameters.
+
+
+Earlier proposed changes of K to 750 or 1000 are far too small. We would still have a tremendous gap between real public pledge and saturation (between the rich founders and everybody else). Raising K by four times its present value will make pledge four times more noticeable and bridge the mentioned gap {fig.3}. Pledge only being somewhat more noticeable won't be enough to change pledge's tarnished reputation. 'Pledge is now 4x more effective' in an article headline and YouTube title will grab the layperson's attention. It will change the narrative from 'pledge has no value' to 'pledge is preferred'. This will all be done while decreasing the rewards paid to saturated pools.
+
+Making pledge more effective could also help solve the problem of exchanges, despite having no stake of their own, making >20% of the blocks. BNP is simply being greedy with their stake. It is more profitable to run their own pledge-less pools than to pay a fee to an SPO. Only a few public pools currently offer greater than baseline rewards, thanks to the imbalance mentioned above. If more pools could offer greater than baseline rewards, exchanges like Binance and Coinbase could choose to make more ADA by delegating to community pools.
```

Comment:
Large exchanges want to:
* Maximise rewards
* Maximise liquidity (minimise pledge and lock-up time)
* Maximise control (eg: preferential submission of their transactions, avoidance of over-saturating stake pools)

Increasing K will make pledge more effective.  Cardano's staking design forces exchanges to choose between increased pledge for increased rewards and reduced pledge for increased liquidity.  This proposal will shift the balance in the pledge direction somewhat and might push exchanges towards community staking.  However, staking to community pools will reduce their control over saturation, and increasing K would make this more difficult to manage.  Implementation of multi-pool delegation could help here?